src/index.js

```diff
@@ -27,3 +27,9 @@ export const murmur3128 = from({
   code: 0x22,
   encode: (input) => bytes.fromHex(mur.x64.hash128(input))
 })
+
+export const murmur364 = from({
```

Comment:
```suggestion
// A special-use 0x22 that truncates 64 bits, specifically for use in the UnixFS HAMT
export const murmur364 = from({
```

Reply:
I don't know if it's special case since multicodec table doesn't even have entry for murmur3-128, but I don't mind including this  comment